charts/nr-ebpf-agent/README.md

```diff
@@ -79,17 +70,29 @@ Options that can be defined globally include `affinity`, `nodeSelector`, `tolera
 
 | Key | Type | Default | Description |
```

Comment:
This table should be autogenerated with `helm-docs` which pulls from `values.yaml`.